3.2/root/usr/share/container-scripts/mongodb/mongodb.conf.template

```diff
@@ -6,13 +6,13 @@ port = ${CONTAINER_PORT}
 # Default: /var/lib/mongodb/data
 dbpath = ${MONGODB_DATADIR}
 
-# Disable data file preallocation. Default: true
+# Disable data file preallocation.
 # only for mounted data directory from older MongoDB server
-noprealloc = ${MONGODB_NOPREALLOC}
+noprealloc = true
```

Comment:
I don't understand this. Why are we removing the argument and then keeping the config?! This should be a separate PR so that we can discuss the effects of that.